lib/cache/cache.go

```diff
@@ -832,7 +838,7 @@ func New(config Config) (*Cache, error) {
 		webTokenCache:                local.NewIdentityService(config.Backend).WebTokens(),
 		windowsDesktopsCache:         local.NewWindowsDesktopService(config.Backend),
 		samlIdPServiceProvidersCache: samlIdPServiceProvidersCache,
-		userGroupsCache:              local.NewUserGroupService(config.Backend),
+		userGroupsCache:              userGroupsSvc,
 		oktaCache:                    oktaSvc,
```

Comment:
Can you make both of these `fooCache` or change `clusterConfigCache` and `samlIdPServiceProvidersCache` into `fooSvc`? 🥺